oca_dependencies.txt

```diff
@@ -0,0 +1,15 @@
+# List the OCA project dependencies, one per line
+# Add a repository url and branch if you need a forked version
+#
+# Examples
+# ========
+#
+# To depend on the standard version of sale-workflow, use:
+# sale-workflow
+#
+# To explicitely give the URL of a fork, and still use the version specified in
+# .travis.yml, use:
+# sale-workflow https://github.com/OCA/sale-workflow
+#
+# To provide both the URL and a branch, use:
+server-ux https://github.com/OCA/server-ux 13.0
```

Comment:
just server-ux is enough, you can remove all the comments I think.

Reply:
also the url

Reply:
@hveficent done

Reply:
@hveficent is it different in v13? until now if it's automatic for OCA repos: https://github.com/OCA/purchase-workflow/blob/12.0/oca_dependencies.txt

Reply:
no, like you said, simple server-ux, for example.

Reply:
@pegonzalezspesol sorry for bothering you with this minor thing, but I think is better to be align in all repos, so can you update the file again to be just "sever-ux"?